components/wpt-runs.html

```diff
@@ -132,7 +132,9 @@
             })
 
         // We flatten into an array of objects so Polymer can deal with them.
-        const flattened = Object.entries(testRuns).map(entry => ({ sha: entry[0], runs: entry[1] }))
+        const flattened = Object.entries(testRuns)
+            .map(entry => ({ sha: entry[0], runs: entry[1] }))
+            .sort(entry => Object.values(entry.runs)[0].created_date)
```

Comment:
Array.prototype.sort takes a function that receives 2 params - item a and item b - and returns `-1`, `0`, or `1`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort